sahi/annotation.py

```diff
@@ -14,6 +14,13 @@
 )
 from sahi.utils.shapely import ShapelyAnnotation
 
+try:
+    from pycocotools import mask as mask_utils
+
+    use_rle = True
+except ImportError:
+    use_rle = False
+
```

Comment:
since `pycocotools` is moved to `requirments.txt` as a direct dependency, I think we change this import procedure, I think the following would be sufficient

```python
from pycocotools import mask as mask_utils
``` 

but @ChristofferEdlund got a point that installing pycoctools on Windows is not straightforward, should we make this a soft dependency rather ? @fcakyon

Reply:
As @fcakyon pointed out, it seems like one can actually install pycocotools on windows right now. But it is still this hurdle to get over:

https://stackoverflow.com/questions/64261546/how-to-solve-error-microsoft-visual-c-14-0-or-greater-is-required-when-inst

Reply:
then it might be better to make pycocotools optional, c ompiler might cause issues to some users.

Reply:
i will handle the required updates 👍

Reply:
> 

@fcakyon, maybe.. Those plots where generated by monitoring the RAM usage of a docker container running on internal models and data. Let me think if there is a good way to do something similar with default SAHI and some good open source images.